var/spack/repos/builtin/packages/zfp/package.py

```diff
@@ -6,26 +6,45 @@
 from spack import *
 
 
-class Zfp(CMakePackage):
-    """zfp is an open source C/C++ library for high-fidelity, high-throughput
-       lossy compression of floating-point and integer multi-dimensional
-       arrays.
+class Zfp(CMakePackage, CudaPackage):
+    """zfp is a compressed format for representing multidimensional
+       floating-point and integer arrays. zfp provides compressed-array
+       classes that support high throughput read and write random access
+       to individual array elements. zfp also supports serial and parallel
+       (OpenMP and CUDA) compression of whole arrays.
```

Comment:
The first paragraph in a package docstring is used by Spack wherever a short description is needed, while the entire docstring is used as a long description. I would thus suggest to start a new paragraph after the first sentence:
```suggestion
    """zfp is a compressed format for representing multidimensional
       floating-point and integer arrays. 
       
       zfp provides compressed-array
       classes that support high throughput read and write random access
       to individual array elements. zfp also supports serial and parallel
       (OpenMP and CUDA) compression of whole arrays.
```

Reply:
Works for me.